Presentation.Web/app/components/data-processing/data-processing-registration-overview.controller.ts

```diff
@@ -208,11 +208,10 @@
                             .withDataSourceName("AgreementConcludedAt")
                             .withTitle("Dato for indgåelse af databehandleraftale")
                             .withId("agreementConcludedAt")
-                            .withStandardWidth(150)
+                            .withStandardWidth(160)
                             .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Date)
                             .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.AgreementConcludedAt))
-                            .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderDate(dataItem.AgreementConcludedAt))
-                            .withInitialVisibility(false))
```

Comment:
Den skal ikke længere være skjult som standard?

Reply:
Det er i hvert fald ikke det vi har aftalt med kunden, men det er muligt at vi to har snakket om det.. nu har jeg bare rettet det i hht aftalen :) 
![image](https://user-images.githubusercontent.com/13116889/94893380-564e4c00-0487-11eb-82ac-7185d932ff2b.png)